storage/store.go

```diff
@@ -238,14 +240,14 @@ var _ multiraft.Storage = &Store{}
 func NewStore(clock *hlc.Clock, eng engine.Engine, db *client.KV, gossip *gossip.Gossip) *Store {
 	s := &Store{
 		StoreFinder: &StoreFinder{gossip: gossip},
-
-		clock:     clock,
-		engine:    eng,
-		db:        db,
-		allocator: &allocator{},
-		gossip:    gossip,
-		closer:    make(chan struct{}),
-		ranges:    map[int64]*Range{},
+		RetryOpts:   defaultRangeRetryOptions,
```

Comment:
Not sure what most defaults should be. `Constant: 2` isn't necessarily the only obvious choice either. I could maybe have a method for creating a retry options struct which takes fewer args, but that's essentially what the composite literal is. I'm not convinced inserting non-Go default values is worth the possible confusion.